pkg/lvm/lvm_util.go

```diff
@@ -444,3 +447,38 @@ func snapshotExists(snapVolumeName string) (bool, error) {
 	}
 	return true, nil
 }
+
+// getVGSize get the size in bytes for given volumegroup name
+func getVGSize(vgname string) string {
+	cmd := exec.Command("vgs", vgname, "--noheadings", "-o", "vg_free", "--units", "b", "--nosuffix")
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		klog.Errorf("failed to list existing volumegroup:%v , %v", vgname, err)
+		return ""
+	}
+	return strings.TrimSpace(string(out))
+}
+
+// getThinPoolSize gets size for a given volumegroup, compares it with
+// the requested volume size and returns the minimum size as a thin pool size
+func getThinPoolSize(vgname, volsize string) string {
+	outStr := getVGSize(vgname)
+	vg_free_size, err := strconv.Atoi(strings.TrimSpace(string(outStr)))
```

Comment:
Atoi will return int, should we use ParseInt(s, 10, 64) here

Reply:
yes, changed it use ParseInt